File: harry/stage6/OurLogic.java
public class OurLogic {
    public static void main(String[] args) {
        // 실행 객체 생성
        Deal deal = new Deal();
        // 구상 커맨드 생성
        IncomeMoneyModule incomeMoneyModule = new IncomeMoneyModule();
        MoneyCommand moneyCommand = new MoneyCommand(incomeMoneyModule, 100000);

        StockModule stockModule = new StockModule();
        StockCommand stockCommand = new StockCommand(stockModule, 10);

        // 커맨드를 로드
        deal.setCommand(moneyCommand);
        deal.setCommand(stockCommand);

        // 로직을 실행
        deal.play();

        // <정상 케이스 로그>
        //        돈을 100000원 입금 합니다.
        //        주식을 10주 입고합니다.

    }
}

File: harry/stage6/ktVer/OurLogic.kt
package ktVer

fun main() {
    // 실행객체 생성
    val deal = Deal()
    // 커맨드 구상 클래스 생성
    var mm = IncomeMoneyModule()
    var sm = StockModule()
    // 100,000 원을 입급하려는 커맨드
    var moneyCommand = MoneyCommand(mm, 100000)
    // 100주를 입고하려는 커맨드
    var stockCommand = StockCommand(sm, 100)

    // 커맨드 세팅
    deal.setCommand(moneyCommand)
    deal.setCommand(stockCommand)

    // play!
    deal.play()

    // <정상케이스로그>
    //    돈을 100000 원 입금합니다.
    //    주식을 100 주 입고 합니다.


}